src/ui/public/doc_table/components/table_row/cell.html

```diff
@@ -13,17 +13,15 @@
   <span class="table-cell-filter">
     <% if (filterable) { %>
       <span
-        ng-click="inlineFilter($event, '+')"
+        ng-click="inlineFilter('<%- column %>', '+')"
```

Comment:
Oh, thanks for pointing that out. In that case I'll defer that change to the major cleanup when we get rid of the lodash templating.